build.gradle

```diff
@@ -448,7 +448,7 @@ subprojects {
         publications {
             nebula(MavenPublication) {
                 if(project.hasProperty('release')) {
-                    signArchives.signatures.each {
+                    signArchives.signatures.findAll({!it.toSign.canonicalPath.contains("deb") && !it.toSign.canonicalPath.contains("rpm")}).each {
```

Comment:
Do we want a story to look at this in the future? I feel like there has to be a way to publish multiple signatures.

Reply:
You can but it requires doing a bunch of crap with classifiers, because we do in fact publish multiples where we are using classifiers (the fat jar). these artifacts are already internally signed, so i felt publishing a completely secondary signature for them was unnecessary. Note that this change would allow any other properly classified artifacts we add later to just work.

Reply:
Fair enough.